ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/LedgerFactory.scala

```diff
@@ -78,6 +79,8 @@ trait ConfigProvider[ExtraConfig] {
   def authService(config: Config[ExtraConfig]): AuthService =
     AuthServiceWildcard
 
+  def interceptors(config: Config[ExtraConfig]): List[ServerInterceptor] = List.empty
```

Comment:
Just to keep it consistent.

```suggestion
  def interceptors(config: Config[ExtraConfig]): List[ServerInterceptor] =
    List.empty
```